PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java

```diff
@@ -83,7 +84,6 @@ public void setListener(PayPalNativeCheckoutListener listener) {
      * This method must be invoked on a {@link PayPalNativeCheckoutClient (Fragment, BraintreeClient)} or
      * {@link PayPalNativeCheckoutClient (FragmentActivity, BraintreeClient)} in order to receive results.
      */
-    @Deprecated
```

Comment:
Ahh whoops actually looks like these methods might have already been deprecated - let's leave the specific methods deprecated if they already were. (Same with constructor)

Reply:
addressed here: bd7d29279c755b3029da1b6181835e43ef6e60bc

Reply:
One more nitpick: the spacing is now off and the constructor above (line 45) can also keep it's deprecated tag

Reply:
addressed here: c10b4a80d8f122d31016be31968650fb5cb72907